content/en/integrations/faq/list-of-api-source-attribute-value.md

```diff
@@ -24,7 +24,7 @@ kind: faq
 |Amazon Cognito|COGNITO|
```

Comment:
Wait, why are these updates being done manually here? I thought we hadn't switched this page to be manual yet. Also, how did you do these updates manually? Won't the prebuild just overwrite them? Or it would have if it weren't disabled right now.